server/evaluations.js

```diff
@@ -33,11 +33,11 @@ exports.setApp = function (app, pool) {
     course = course.replace(/[^-a-z0-9 \/]/g , "");
     var professor = searchParams["professor"]
     professor = professor.replace(/[^-a-z0-9 \/]/g , "");
-    var queryString = "SELECT comments FROM dopeevals WHERE";
+    var queryString = "SELECT * FROM dopeevals WHERE";
     if (course.length == 0) queryString += util.format(" professor LIKE '%%%s'", professor);
     else if (professor.length == 0) queryString += util.format(" course LIKE '%%%s%%'", course);
     else queryString += util.format(" professor LIKE '%%%s' AND course LIKE '%%%s%%'", professor, course);
-    queryString += " AND comments IS NOT NULL ORDER BY adddate DESC;";
+    queryString += "AND GRADES <> 0 ORDER BY adddate DESC;";
```

Comment:
What's the reasoning for removing the leading space? I thought that would be needed to construct the query, to make sure it doesn't look like `...course LIKE 'environmental'AND GRADES...`. Also, I think grades should be lowercase for style consistency.

Reply:
Thanks for the catch! I added the grades filter last-minute to filter out dummy reviews, so must have missed the inconsistency when I published this. Fixed in next revision.